lua/onedarkpro/config.lua

```diff
@@ -35,6 +36,7 @@ local defaults = {
         lua = true,
         markdown = true,
         php = true,
+        printf = true,
```

Comment:
Propose we remove this and merge all highlights within the c filetype

Reply:
I am thinking that this would also have some use in C++. I will leave that decision up to you. It is only one highlight from what I can tell, so easy to copy-paste.